tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -321,13 +455,22 @@ platform_tests/api/test_psu.py::TestPsuApi::test_power:
     reason: "Unsupported platform API"
     conditions:
       - "asic_type in ['mellanox', 'cisco-8000'] or (asic_type in ['barefoot'] and hwsku in ['newport']) or platform in ['armhf-nokia_ixs7215_52x-r0']"
-
+      - "hwsku in ['Celestica-DX010-C32']"
```

Comment:
@assrinivasan If add new conditions here, the operator used is AND, I think for platform_tests/api/test_psu.py::TestPsuApi::test_power, it will not work expected.

In your subject, I think your purpose should xfail these cases, but you skip all cases, should it be xfail? Is the following better?

```
platform_tests/api/test_psu.py::TestPsuApi::test_power:
  skip:
    reason: "Unsupported platform API"
    conditions:
      - "asic_type in ['mellanox', 'cisco-8000'] or (asic_type in ['barefoot'] and hwsku in ['newport']) or platform in ['armhf-nokia_ixs7215_52x-r0']"
  xfail:
    reason: "Celestica known issue"
    conditions:
      - "hwsku in ['Celestica-DX010-C32']"
      - https://github.com/sonic-net/sonic-mgmt/issues/6512
```